Makefile

```diff
@@ -72,28 +72,3 @@ clean-dev:
 test:
 	npm run test
 	./vendor/phpunit/phpunit/phpunit -c phpunit.xml
-
-build-tarball:
```

Comment:
Don't we need this? I think some things might have slipped through the conflict handling.

Reply:
There are no test but I think there should be simple units tests. for now the test script is just an echo of "No tests implemented"

Reply:
I was mainly referring to the build-tarball portion being removed

Reply:
npm run test is a placeholder. I think we should add tests in the future. For now it is just echoing a message.

Reply:
I'm still not referring to the `make test`, the entire `make build-tarball` is being removed here, and is needed for the prerelease workflow.

Reply:
my bad, I thought I fixed that with git reset but no. This commit should have fixed that. I have confirmed that `make build-tar` it is working on my latest local commit.